src/sage/graphs/generators/basic.py

```diff
@@ -1149,6 +1149,89 @@ def LadderGraph(n):
     return G
 
 
+def MoebiusLadderGraph(n):
+    r"""
+    Return a Möbius ladder graph with `2n` nodes
+
+    A Möbius ladder graph of order `2n` is a ladder graph of the same order
+    that is connected at the ends with a single twist, i.e., a ladder graph
+    bent around so that top meets bottom with a single twist. Alternatively,
+    it can be described as a single cycle graph (of order `2n`) with the
+    addition of edges (called `rungs`) joining the antipodal pairs of nodes.
+    Also, note that the Möbius ladder graph ``graphs.MoebiusLadderGraph(n)`` is
+    precisely the same graph as the circulant graph
+    ``graphs.CirculantGraph(2 * n, [1, n])``.
+
+    PLOTTING:
+
+    Upon construction, the position dictionary is filled to override the
+    spring-layout algorithm. By convention, each Möbius ladder graph will be
+    displayed with the first (0) node at the top, with the rest following in a
+    counterclockwise manner.
+
+    INPUT:
+
+    - ``n`` -- a non negative integer; number of nodes is `2n`
+
+    OUTPUT:
+
+    - ``G`` -- a Möbius graph of order `2n`; note that a :class:`ValueError` is
+      returned if ``n < 0``
+
+    EXAMPLES:
+
+    Construct and show a Möbius ladder graph with 26 nodes::
+
+        sage: g = graphs.MoebiusLadderGraph(13)
+        sage: g.show()  # long time  # needs sage.plot
```

Comment:
the `# needs sage.plots` should start at columns 88. This is a convention to improve readability.